pkg/ucp/api/v20231001preview/awsplane_conversion_test.go

```diff
@@ -41,7 +41,7 @@ func Test_AWSPlane_ConvertVersionedToDataModel(t *testing.T) {
 					TrackedResource: v1.TrackedResource{
 						ID:       "/planes/aws/aws",
 						Name:     "aws",
-						Type:     "System.AWS/planes",
+						Type:     datamodel.AWSPlaneResourceType,
```

Comment:
I think this requires a rebase when the other PR gets merged in.

Reply:
That's the idea 👍